src/transformers/modeling_utils.py

```diff
@@ -1747,6 +1747,9 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
                 quantization works well for values of magnitude ~5, but beyond that, there is a significant performance
                 penalty. A good default threshold is 6, but a lower threshold might be needed for more unstable models
                 (small models, fine-tuning).
+            no_load_in_8bit_modules (`List[str]`, *optional*, defaults to `None`):
+                An explicit list of the modules that we do not want to convert in 8-bit. This is useful for models such as
+                Jukebox that has several heads in different places and not necessarly at the last position.
```

Comment:
```suggestion
                Jukebox that has several heads in different places and not necessarily at the last position.
```

Reply:
Fixed in 01a4c0cea9d2c2ad8fd3c85afbdb274510d91344 !